packages/actor-http-fetch/test/ActorHttpFetch-test.ts

```diff
@@ -361,5 +372,60 @@ describe('ActorHttpFetch', () => {
       })).rejects.toThrow('foo');
       jest.useRealTimers();
     });
+
+    it('should work with a large timeout including body if there is no body', async() => {
+      await expect(actor.run({
+        input: <Request> { url: 'NOBODY' },
+        context: new ActionContext({ [KeysHttp.httpTimeout.name]: 100_000, [KeysHttp.httpTimeoutOnBody.name]: true }),
+      })).resolves.toMatchObject({ status: 404 });
+    });
+
+    it('should work with a large timeout including body if the body is consumed', async() => {
+      const customFetch = jest.fn(async(_, _init) => {
+        const body = Readable.from('foo');
+        return Promise.resolve({
+          body,
+        });
+      });
+      const response = await actor.run({
+        input: <Request> { url: 'https://www.google.com/' },
+        context: new ActionContext({
+          [KeysHttp.fetch.name]: customFetch,
+          [KeysHttp.httpTimeout.name]: 100_000,
+          [KeysHttp.httpTimeoutOnBody.name]: true,
+        }),
+      });
+      const body = <Readable><any> response.body;
+      for await (const chunk of body) {
+        // We just want to consume everything
+      }
+    });
+
+    // FIXME: does not work
+    // it('should work with a large timeout including body if the body is not consumed', async() => {
+    // await expect(actor.run({
+    //     input: <Request> { url: 'https://www.google.com/' },
+    //     context: new ActionContext({
+    //       [KeysHttp.httpTimeout.name]: 100_000,
+    //       [KeysHttp.httpTimeoutOnBody.name]: true
+    //     }),
+    // })).resolves.toMatchObject({ status: 200 });
+    // });
```

Comment:
This test does not work yet because there is now way to act if a steam is not read but just "dropped" by the code and GCed.